atom/common/api/atom_bindings.cc

```diff
@@ -68,6 +69,38 @@ v8::Local<v8::Value> GetSystemMemoryInfo(v8::Isolate* isolate,
   return dict.GetHandle();
 }
 
+v8::Local<v8::Value> GetCPUUsage(v8::Isolate* isolate) {
+  std::unique_ptr<base::ProcessMetrics> metrics(
+      base::ProcessMetrics::CreateCurrentProcessMetrics());
+
+  mate::Dictionary dict = mate::Dictionary::CreateEmpty(isolate);
+  int processor_count = base::SysInfo::NumberOfProcessors();
+  dict.Set("percentCPUUsage",
+           metrics->GetPlatformIndependentCPUUsage() / processor_count);
+  dict.Set("idleWakeupsPerSecond", metrics->GetIdleWakeupsPerSecond());
+
+  return dict.GetHandle();
+}
+
+v8::Local<v8::Value> GetIOCounters(v8::Isolate* isolate) {
+  std::unique_ptr<base::ProcessMetrics> metrics(
+      base::ProcessMetrics::CreateCurrentProcessMetrics());
+  base::IoCounters io_counters;
+  const bool got_counters = metrics->GetIOCounters(&io_counters);
+  mate::Dictionary dict = mate::Dictionary::CreateEmpty(isolate);
+
+  if (got_counters) {
```

Comment:
I think it is fine to just right this as:

```cpp
if (metrics->GetIOCounters(&io_counters))
```

And then remove the `got_counters` variable.